source/helpers/fetch-dom.ts

```diff
@@ -1,9 +1,10 @@
+/* eslint-disable no-redeclare */
```

Comment:
Why was this enabled? 🤔 

It's disabled here

https://github.com/xojs/eslint-config-xo-typescript/blob/a9481df0e9b382287ce94fd3ec394bf9d59dae51/index.js#L374